test/jdk/com/sun/net/httpserver/HttpsParametersClientAuthTest.java

```diff
@@ -220,9 +223,11 @@ public void configure(final HttpsParameters params) {
                             throw ioe;
                         }
                         // verify it failed due to right reason
-                        Throwable cause = ioe;
+                        Throwable cause = ioe.getCause();
```

Comment:
shouldn't we start with `cause = ioe` like before? Or was this done to make sure we only look at the nested exception on windows?

Reply:
Hello Daniel, yes this is intentional - The change to start with `ioe.getCause()` was prompted due to the Windows check, but ultimately since we are only always interested in the underlying cause and not the top level `IOException`, this change is thus relevant for other OSes as well. I have run the tests to make sure it's only the underlying cause (and not the top level one) which has the correct expected exception that this test checks for.